cmake/linker/ld/linker_libraries.cmake

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Do not specify default link libraries when targeting host (native build).
+if(NOT CONFIG_NATIVE_BUILD)
+  set_linker_property(NO_CREATE TARGET linker PROPERTY c_library    "-lc")
+  set_linker_property(NO_CREATE TARGET linker PROPERTY rt_library   "-lgcc")
```

Comment:
Yeah, it would be nice to see the libgcc path defined here so that it was tied with the library name. I haven't dug into how that value is computed though.

Reply:
it's probably nice to have it with a comment regarding why it's empty at this location and where it is being filled.

Reply:
done.